CHANGELOG.md

```diff
@@ -64,13 +65,6 @@ All notable, unreleased changes to this project will be documented in this file.
 - Fix incorrect cart badge location - #3786 by @jxltom
 - Fix storefront styles after bootstrap is updated to 4.3.1 - #3753 by @jxltom
 - Fix logo size in different browser and devices with different sizes - #3722 by @jxltom
-- Rename dumpdata file `db.json` to `populatedb_data.json` - #3810 by @maarcingebala
```

Comment:
@maarcingebala Are these changelog deleted accidentally?

Reply:
Oops, yes, I was trying to fix changelog conflict but something went wrong

Reply:
Could you fix the Changelog and rebase this branch with master?

Reply:
Sure, done